include/QBDI/Platform.h

```diff
@@ -21,16 +21,16 @@
 #include "Config.h"
 
 #ifdef QBDI_PLATFORM_WINDOWS
-# define _QBDI_UNREACHABLE() __assume(0)
```

Comment:
Why removing _QBDI_UNREACHABLE ?

Reply:
I replace all the ``_QBDI_UNREACHABLE()`` with switch where all branch return.

However, the compilation on windows returns some ``warning C4715: 'XXX': not all control paths return a value``. I revert the changes to fix the warning.